_activities/collisions.md

```diff
@@ -4,12 +4,12 @@ layout: activity
 # Content
 #
 title: "Collisions"
-title_fr: "Collisions"
```

Comment:
This file should not have been edited (it is being reverted to a previous version here.)